### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by tmhastings**
- Update index.md file to match the overview file previously edited.
The detailed explanation under "What is Trio?" was NOT altered as this information should be on the home page. The rest of the information is updated to match what is located elsewhere in the documentation.

- links corrected
- orphaned file reference removed

### CODE REVIEW

**Comment by MikePlante1** [comment]
Looks like these links didn't format correctly:
Screenshot 2024-05-09 at 4 42 41 PM

Guessing it's because it's markdown inside of html?

**Comment by marionbarker** [comment]
The links are not correct, here's an example:

```
https://github.com/nightscout/Trio-docs/blob/Operate-Folder-Updates/docs/EN/settings/configuration/carbratios.md#insulin-carbohydrate-ratio
```

You might mean to use `dev` instead of `Operate-Folder-Updates` for now, with a plan to migrate to `main` soon.
Or perhaps you plan to start with `main` and `dev` at the same version (commit) when you announce for beta testers, in which case, change the links to point to `main.

Either approach works. But please someone needs to check all the links in the entire docs for incorrect links of this type.

**Comment by MikePlante1** [comment]
> The links are not correct, here's an example:
> 
> ```
> https://github.com/nightscout/Trio-docs/blob/Operate-Folder-Updates/docs/EN/settings/configuration/carbratios.md#insulin-carbohydrate-ratio
> ```
> 
> You might mean to use `dev` instead of `Operate-Folder-Updates` for now, with a plan to migrate to `main` soon. Or perhaps you plan to start with `main` and `dev` at the same version (commit) when you announce for beta testers, in which case, change the links to point to `main.
> 
> Either approach works. But please someone needs to check all the links in the entire docs for incorrect links of this type.

Better solution, I think, would be to make the links relative instead of absolute, that way it doesn't link to a specific branch of docs

**Comment by tmhastings** [comment]
I corrected them with the last 3 commits using relative links and removing the dropdowns. I checked that it linked to the correct locations. Please verify it does the same for you (@MikePlante1 or @marionbarker) and submit a review.

**Comment by MikePlante1** [comment]
@tmhastings Links didn't work for me. You should exclude the `/docs/en/` part off. It'll change for different languages anyway, whenever those are added in the future.

I'd also suggest putting the acronym in parenthesis following the full term.

diff

```md
- [Carb Ratio (CR)](settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
- [Insulin Sensitivity Factor (ISF)](settings/configuration/insulinsensitivities.md#insulin-sensitivity-factor)
- [Basal Rate](settings/configuration/basalprofile.md#basal-profile)
```

**Comment by tmhastings** [comment]
> @tmhastings Links didn't work for me. You should exclude the `/docs/en/` part off. It'll change for different languages anyway, whenever those are added in the future.
> 
> I'd also suggest putting the acronym in parenthesis following the full term.
> 
> diff
> ```
> - [Carb Ratio (CR)](settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
> - [Insulin Sensitivity Factor (ISF)](settings/configuration/insulinsensitivities.md#insulin-sensitivity-factor)
> - [Basal Rate](settings/configuration/basalprofile.md#basal-profile)
> ```

Done!

**Comment by tmhastings** [comment]
I also updated the work in progress to reflect the current state of Trio, warn general users against using a beta version, and instruct beta testers on what to do should they locate an error in the documentation. Also added links for quicker referencing.

**Comment by marionbarker** [comment]
I don't know how sphinx works but every other website does not start a relative link with a slash "/".

For example, looking at the raw version of index.md, I see this:
```
Information on [Carb Ratio (CR)](/settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
```

The location of index.md in the folder structure is `docs/EN/index.md`
The location of  carbratios.md in the folder structure is `docs/EN/settings/configuration/carbratios.md`

I would expect the link should be formatted like this: 
```
Information on [Carb Ratio (CR)](settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
```

or 

```
Information on [Carb Ratio (CR)](../settings/configuration/carbratios.md#insulin-carbohydrate-ratio)
```

**Comment by tmhastings** [comment]
Ok. Hopefully they work now? 😁

**Review by marionbarker — DISMISSED** [review]
That fixed the links. (I also did a global search and found no more instances of `(/relative-path` in the *.md files.

I don't know if you want to add a commit to remove `Other AIDs <./resources/alternative.md>` because you already deleted the file or open a separate PR.

**Comment by tmhastings** [comment]
I can include them in PR #35 as multiple instances of that occurring were indicated by the Loop and Learn team during their review.

**Comment by tmhastings** [comment]
> That fixed the links. (I also did a global search and found no more instances of `(/relative-path` in the *.md files.
> 
> I don't know if you want to add a commit to remove `Other AIDs <./resources/alternative.md>` because you already deleted the file or open a separate PR.

I will include it in this PR and tag for your review of the changes.

Edit: @marionbarker ready for final review after removing orphaned link to alternative.md

**Review by marionbarker — DISMISSED** [review]

File: docs/EN/index.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 :::{admonition} Work-In-Progress
 :class: caution
 
-**PLEASE NOTE THIS DOCUMENTATION IS STILL BEING WRITTEN BY OUR AWESOME COMMUNITY. UNTIL IT IS FINISHED, PLEASE DO NOT SET UP YOUR TRIO BASED ON THIS DOCUMENTATION ALONE. CONTACT THE COMMUNITY ON DISCORD OR FACEBOOK TO MAKE SURE YOU GET IT RIGHT AND STAY SAFE**
+**GENERAL USERS** This documentation is still under construction during the beta release process. It should not be the main source of information until the main branch of Trio is released. If you have any questions, contact the community on [Discord](https://discord.gg/fCY5svg4) and/or [Facebook](https://www.facebook.com/groups/1351938092206709). Be advised that it is not recommended that general users use the beta version unless they are prepared to update quickly and frequently and report any issues found.
```
Comment (MikePlante1) [inline docs/EN/index.md]: Discord link is broken.

Comment (tmhastings) [reply docs/EN/index.md]: Updated with working, Trio Official-generated, set to never expire with unlimited uses link: https://discord.gg/FnwFEFUwXE

File: docs/EN/index.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 :::{admonition} Work-In-Progress
 :class: caution
 
-**PLEASE NOTE THIS DOCUMENTATION IS STILL BEING WRITTEN BY OUR AWESOME COMMUNITY. UNTIL IT IS FINISHED, PLEASE DO NOT SET UP YOUR TRIO BASED ON THIS DOCUMENTATION ALONE. CONTACT THE COMMUNITY ON DISCORD OR FACEBOOK TO MAKE SURE YOU GET IT RIGHT AND STAY SAFE**
+**GENERAL USERS** This documentation is still under construction during the beta release process. It should not be the main source of information until the main branch of Trio is released. If you have any questions, contact the community on [Discord](https://discord.gg/FnwFEFUwXE) and/or [Facebook](https://www.facebook.com/groups/1351938092206709). Be advised that it is not recommended that general users use the beta version unless they are prepared to update quickly and frequently and report any issues found.
+**BETA TESTERS** Because this documentation is still under development, if you notice any discrepancies, errors, or edits that need to be made, please submit an [Issue](https://github.com/nightscout/trio-docs/issues/new/choose) on GitHub.
```
Comment (MikePlante1) [inline docs/EN/index.md:6]: If you want `BETA TESTERS` on a new line, add `<br>` to the beginning of line 5

Screenshot 2024-06-19 at 2 31 16 PM

**Review by MikePlante1 — APPROVED** [review]

**Comment by tmhastings** [comment]
Replaced with PR #48

**Pull request closed without merging**